src/api.js

```diff
@@ -252,6 +252,11 @@ var api = {
       return Promise.reject(err);
     });
   },
+
+  /**
+   * Deprecated. Call `listFirebaseProjects()` from `./management/project.ts` instead
+   * TODO: remove this function after deprecating `list` command
```

Comment:
What do you think about putting a `logger.debug` in the body of `getProjects` that does something like

```typescript
logger.debug(`[WARNING] ${(new Error("getProjects() is deprecated - update the implementation")).stack}`)
```

It won't muddy up the output normally, but if something is still using `getProjects` it would become evident in the `--debug` or `firebase-debug.log` output. What do you think?

Reply:
This is great. Updating